bin/serve

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+// example: serve folder --port 8000 --proxy /api=https://api.razorpay.com
+const http = require('http');
+const argv = require('../argv');
+const path = require('path');
+const proxyServer = require('http-proxy').createProxyServer();
+
+const root = path.resolve(argv._[0] || argv.root || '')
+const ecstatic = require('ecstatic')({ root });
+
+const getProxies = () => {
+  let proxies;
+  if (Array.isArray(argv.proxy)) {
+    proxies = argv.proxy;
+  } else if (argv.proxy) {
+    proxies = [argv.proxy];
+  } else {
+    proxies = [];
+  }
+  return proxies;
+}
+
```

Comment:
```suggestion
const getProxies() => {
  return [].concat(argv.proxy).filter(Boolean);
}
```

Reply:
@sidx1024 Why do you need the filter?

Reply:
If `argv.proxy` is null then it returns an empty array, not an array with null in it.